### PR TITLE
users prespective:the front view documentation

### PR DESCRIPTION
**Pull request by smriti1313**
Fixes issue: #12 

- Created a `Start` folder where files like License,requirements.txt are added so as to have a better look of repo.
- The requested markdown format of user perspective:the front end documentation is added in `Start` folder with file named `readme2.md`. 
- Since it was not specified where to add this documentation, so I haven't made any changes in Original readme.

image
As you can see, here can be alignment issue but it will change according to the size of space available or window size.

If there are any further changes required then do let me know. 

Thank you

### CODE REVIEW

File: Start/readme2.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+## 4.3.3 &ensp; Users Perspective: The Front View
```
Comment (IIITM-Jay) [inline Start/readme2.md:1]: @smriti1313 ```&ensp``` does not come under markdown format... it's a complete html file :disappointed:

File: Start/readme2.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The various aspects an users expects from the system and which is my one of the main ideal outcomes of the project can be viewed from the following figure:
+
+<p align=center>
+<img src="Start\user perspective.png" height=390px width=390px>
+</p>
```
Comment (IIITM-Jay) [inline Start/readme2.md]: @smriti1313 Again html tags. Actually we want this file to be written in a complete markdown format without any inclusion of html tags and attributes. :worried:

File: Start/readme2.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+<br>
+&ensp; The above figure portrays the 
+exibilities and accessibilities provided to the users in order to
+perform various operations on the properties included in the databases. These are:
+
+<br>
+<br>
+<b>&ensp;&ensp;
+1. <i>View:</i>
```
Comment (IIITM-Jay) [inline Start/readme2.md:16]: @smriti1313 ```<i>View:</i> ``` This shows that you have to explore more on markdown formats.

**Review by IIITM-Jay — CHANGES_REQUESTED** [review]
@smriti1313 It is requested to follow the strict markdown rules and formats because it is fundamental for any github repository :new:

File: Start/readme2.md (hunk 3 above)
Comment (smriti1313) [reply Start/readme2.md:16]: Hi, Sorry. I will solve all this by tomorrow.

**Comment by smriti1313** [comment]
Hi, I have made the changes. I am confused with the alignment and size of the image since markdown does not provide any such feature. If you have any idea then please help.

**Comment by IIITM-Jay** [comment]
@samir-0711 Smriti can you align the photo in center...(using html.. it's okay) and remove that caption and add the same caption by deleting ``Fig 4.4 ``

**Comment by smriti1313** [comment]
Hi, @IIITM-Jay I have made the changes you asked for. Kindly review it.
Thank you

File: Start/readme2.md [hunk 4]
```diff
@@ -1,9 +1,15 @@
 ## 4.3.3   Users Perspective: The Front View
```
Comment (IIITM-Jay) [inline Start/readme2.md:1]: @smriti1313 SMIRITI remove the heading 4.3.3

File: Start/readme2.md [hunk 5]
```diff
@@ -1,9 +1,15 @@
...
 The various aspects an users expects from the system and which is my one of the main ideal outcomes of the project can be viewed from the following figure:
 
-![](https://github.com/smriti1313/Property-management-System-Django-Full-Stack-/blob/main/Start/user%20perspective.png)
+<p align=center>
+<img src="user_perspective.png" height=400px >
+</p>
+<br>
+<p align="center">
+  Figure 4.4: Operations offered to the users
+</p>
```
Comment (IIITM-Jay) [inline Start/readme2.md:10]: @smriti1313 smiriti that's what I told you in previous comment remove ``figure 4.4`` while keep the caption as it is

**Review by IIITM-Jay — CHANGES_REQUESTED** [review]
@smriti1313 Smriti go thorough the last minute changes as I mentioned accordingly.

File: Start/readme2.md (hunk 5 above)
Comment (smriti1313) [reply Start/readme2.md]: Hi, apologies for the same. Changes has been made.

**Comment by smriti1313** [comment]
Hi, 
@IIITM-Jay @Nandini-Jha Can you please review my PR?

Apologies for the trouble

Thank you

**Comment by IIITM-Jay** [comment]
@all-contributors please add @smriti1313  for content

**Comment by IIITM-Jay** [comment]
@all-contributors please add @smriti1313  for content

**Pull request merged**